src/main/settings_manager.ts

```
@@ -5,7 +5,7 @@
 
 // Dynamically build the Settings type from ENV_KEYS and APP_SETTINGS
 export type Settings = {
-  [K in EnvKey]?: string;
+  APIKEY: Record<string, string>
```

ESLint

[error] Insert `;`

(prettier/prettier)